practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -65,31 +69,14 @@ const Analytics = () => {
   };
 
   // Fetch data from the server when the component mounts
-  useEffect(() => {
-    const getAllCustomers = async () => {
-      setIsLoading(true);
-      const data = await getAllCustomerService();
-      if (data instanceof Error) {
-        setIsError(data.message);
-        toast.error("Having some error. Please try again!", {
-          position: "top-right",
-          autoClose: 5000,
-          hideProgressBar: false,
-          closeOnClick: true,
-          pauseOnHover: true,
-          draggable: true,
-          progress: undefined,
-          theme: "dark",
-        });
-      } else {
-        setCustomers(data);
-        setIsError(null);
-      }
-      setIsLoading(false);
-    };
-
-    getAllCustomers();
-  }, []);
+  const {
+    data: customers,
```

Comment:
where is define customers?

Reply:
I don't define customers, I just use it like example in SWR docs
It use destructuring to asign value
![image](https://github.com/datdragon777/React-Training/assets/74806826/a94edff6-af42-4f02-a6ec-3fd37b085a1d)